spec/support/action_requires_sign_in_helpers.rb

```diff
@@ -0,0 +1,7 @@
+module ActionRequiresSignInHelpers
+  def requires_signed_in_user_to(&block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used. You can also write as `requires_signed_in_user_to(*)` if you want the method to accept any arguments but don't care about them.